ml/jml/testing/weighted_training_test.cc

```diff
@@ -64,66 +64,77 @@ BOOST_AUTO_TEST_CASE( weighted_training_test )
 
     filestream.close(); 
 
-    string cmd1 = "make classifier_training_tool; build/x86_64/bin/classifier_training_tool " + filename1 + " -c " + configname + " -n glz glz.link_function=linear glz.ridge_regression=false -E 1 -p 1 -o " + modelname1 + " glz.verbosity=10  -W LABEL/F,ExampleId/F -z ExampleId";
+    vector<string> probabilizer = {"-p 3", "-p 1 -Q 1"};
+    for(int probIdx=0; probIdx<probabilizer.size(); probIdx++) {
+        cout << ML::format(" ----------------------- TESTING WITH PROBABILIZER IN MODE %s (%s) "
+                "-----------------------", probabilizer[probIdx], (probIdx == 0 ? "off" : "on")) << endl;
 
-    string cmd2 = "build/x86_64/bin/classifier_training_tool " + filename2 + " -c " + configname + " -n glz glz.link_function=linear glz.ridge_regression=false -E 1 -p 1 -o " + modelname2 + " glz.verbosity=10  -W LABEL/F,ExampleId/F,WEIGHT/V -z ExampleId -z WEIGHT";
+        // weights: LABEL, ExampleId
+        // ignore: ExampleId
+        string cmd1 = "make classifier_training_tool; build/x86_64/bin/classifier_training_tool " + filename1 + " -c " + configname + " -n glz glz.link_function=linear glz.ridge_regression=false -E 1 "+probabilizer[probIdx]+" -o " + modelname1 + " glz.verbosity=10  -W LABEL/F,ExampleId/F -z ExampleId";
```

Comment:
We should remove the "make" step, and instead make $(TESTS)/weighted_training_test depend on $(BIN)/classifier_training_tool in the makefile.